config/initializers/config.rb

```diff
@@ -0,0 +1,49 @@
+Config.setup do |config|
+  # Name of the constant exposing loaded settings
+  config.const_name = 'Settings'
```

Comment:
https://github.com/DFE-Digital/manage-courses-backend/blob/214f6da771de448408b8c0b2aa24eea9945a7c83/config/initializers/config.rb#L24

Reply:
Wrong var name, it's `const_name` not `env_prefix`.